docs/book/src/development/development.md

```diff
@@ -195,6 +196,113 @@ and create the environment by running:
 ./hack/ci/create_devstack.sh
 ```
 
+##### DevStack
+
+Here's a few notes to setup a DevStack environment and debug ressources (tested on `m3.small` from Equinix Metal: https://deploy.equinix.com/product/servers/m3-small/)
+
+###### Server side
+
+As a root user, install and configure DevStack:
+
+```
+# useradd -s /bin/bash -d /opt/stack -m stack
+# chmod +x /opt/stack
+# echo "stack ALL=(ALL) NOPASSWD: ALL" | tee /etc/sudoers.d/stack
+# sudo -u stack -i
+$ git clone https://opendev.org/openstack/devstack
+$ cd devstack
+$ cat > local.conf <<EOF
+[[local|localrc]]
+ADMIN_PASSWORD=!!! CHANGE ME !!!
+DATABASE_PASSWORD=\$ADMIN_PASSWORD
+RABBIT_PASSWORD=\$ADMIN_PASSWORD
+SERVICE_PASSWORD=\$ADMIN_PASSWORD
+
+GIT_BASE=https://opendev.org
+# Enable Logging
+LOGFILE=$DEST/logs/stack.sh.log
+VERBOSE=True
+LOG_COLOR=True
+enable_service rabbit
+enable_plugin neutron $GIT_BASE/openstack/neutron
+# Octavia supports using QoS policies on the VIP port:
+enable_service q-qos
+enable_service placement-api placement-client
+# Octavia services
+enable_plugin octavia $GIT_BASE/openstack/octavia master
+enable_plugin octavia-dashboard $GIT_BASE/openstack/octavia-dashboard
+enable_plugin ovn-octavia-provider $GIT_BASE/openstack/ovn-octavia-provider
+enable_plugin octavia-tempest-plugin $GIT_BASE/openstack/octavia-tempest-plugin
+enable_service octavia o-api o-cw o-hm o-hk o-da
+# Cinder
+enable_service c-api c-vol c-sch
+# Tempest
+enable_service tempest
+EOF
+$ ./stack.sh
+```
+
+If you want to enable web-download (i.e import images from URL):
+```
+# /etc/glance/glance-api.conf
+show_multiple_locations = True
+
+# ./horizon/openstack_dashboard/defaults.py
+IMAGE_ALLOW_LOCATIONS = True
+
+# /etc/glance/glance-image-import.conf
+[image_import_opts]
+image_import_plugins = ['image_decompression']
+
+$ sudo systemctl restart devstack@g-api.service apache2
+```
+
+With this dev setup, it might be useful to enable DHCP for the public subnet:
+Admin > Network > Networks > `public` > Subnets > `public-subnet` > Edit Subnet > Subnet Details > :ballot_box_with_check: Enable DHCP + Add DNS
```

Comment:
Where's that done?

Reply:
Worth to say that. But in general I'd prefer doing that from the CLI instead. I'm guessing it's `enable_dhcp` option on the subnet?